targets/TARGET_NORDIC/TARGET_NRF5/us_ticker.c

```diff
@@ -556,82 +454,89 @@ static void register_next_tick()
     __enable_irq();
```

Comment:
not part of these changes but that is the fun of a review - you see so much more than you should or with to :? should this be a critical section?